ldap/ldap.go

```diff
@@ -104,8 +109,8 @@ func (opts CreationOpts) validate() error {
 	}
 
 	if opts.UserCache == nil {
-		if opts.PutCache == nil || opts.GetCache == nil {
-			catcher.Add(errors.New("PutCache and GetCache must not be nil"))
+		if opts.PutCache == nil || opts.GetCache == nil || opts.ClearCache == nil {
```

Comment:
Will this validation make this hard to deploy? ClearCache will not yet be set.

Reply:
Noop